### DISCUSSION TITLE
16.2 Homekit Architecture

### DISCUSSION THREAD

**Discussion by mikwayhk** [RavenSystem/esp-homekit-devices#1927] [Q&A] [answered]
Would the new architecture impact on devices that using Raven or HAA? I'm afraid that causing problems after upgrade.

**Comment by RavenSystem** [comment, marked as answer]
HAA is fully compatible with Apple Home New Architecture introduced in iOS 16.2. To avoid issues, update HAA to last version before migrating to the new architecture.
